pydfc/dfc_methods/sliding_window.py

```diff
@@ -95,39 +96,28 @@ def calc_MI(self, X, Y):
         return MI
 
     def FC(self, time_series):
-
-        if self.params["sw_method"] == "GraphLasso":
-            # Standardize the data (zero mean, unit variance for each feature)
-            mean = np.mean(time_series, axis=1, keepdims=True)
-            std = np.std(time_series, axis=1, keepdims=True)
-            time_series_standardized = np.where(std != 0, (time_series - mean) / std, 0)
-            model = GraphicalLasso(alpha=self.graphical_lasso_alpha_)
-            model.fit(time_series_standardized.T)
-            # the covariance matrix will equal the correlation matrix
+        # Graphical Lasso
+        if self.params['sw_method']=='GraphLasso':
+            model = GraphicalLassoCV()
+            model.fit(time_series.T)
```

Comment:
This part shouldn't change. It's probably because you didn't merge the final version before making the changes.